packages/sync-actions/src/projects-actions.js

```diff
@@ -27,3 +38,37 @@ export function actionsMapBase(diff, oldObj, newObj, config = {}) {
     shouldOmitEmptyString: config.shouldOmitEmptyString,
   })
 }
+
+export const actionsMapBusinessUnit = (diff, oldObj, newObj) => {
+  const { businessUnits } = diff
+  if (!businessUnits) {
+    return []
+  }
+
+  return buildBaseAttributesActions({
+    actions: myBusinessUnitActionsList,
+    diff: businessUnits,
+    oldObj: oldObj.businessUnits,
+    newObj: newObj.businessUnits,
+  })
+}
+
+export function actionsMapCustomer(diff, oldObj, newObj) {
+  const { searchIndexing } = diff
+
+  if (!searchIndexing) {
+    return []
+  }
+
+  const { customers } = searchIndexing
+  if (!customers) {
+    return []
+  }
```

Comment:
That means there is sth wrong with doc ref: [SearchIndexingConfiguration](https://docs.commercetools.com/api/projects/project#searchindexingconfiguration), right?

Reply:
I opened an issue to add the missing field: https://github.com/commercetools/commercetools-docs/issues/5506